src/js/select2/dropdown/attachBody.js

```diff
@@ -32,6 +32,15 @@ define([
           self._resizeDropdown();
         });
       }
+
+	  function repositionOnSelectionChange() {
```

Comment:
The tabbing appears to be off here, as it looks like you are using a combination of spaces and tabs.